### ISSUE TITLE
ip route: support deleting routes

### ISSUE THREAD

**Issue by hugelgupf** [u-root/u-root#1370]
Support for both

- deleting individual routes
- clearing all routes

is really needed.

**Comment by 10000TB** [comment]
does this correspond to iproute flush  ? would this https://github.com/shemminger/iproute2/blob/master/ip/iproute.c#L1691 be a good reference for implementation

**Comment by hugelgupf** [comment]
Yeah, that looks about right. I would choose to perhaps only implement the "table", "dev", and "protocol" filter mechanisms there. (also, just not having any filters at all!)

**Comment by 2SpaceMasterRace** [comment]
Hi @hugelgupf, this looks like something I could work on

is it cool if I get assigned to this so I can get started working

**Comment by hugelgupf** [comment]
I think this might already be supported, we just forgot to close the issue

**Comment by 2SpaceMasterRace** [comment]
Ah too bad thanks anyways

**Comment by hugelgupf** [comment]
Fixed by #1915
